openslides/users/static/templates/users/group-list.html

```diff
@@ -0,0 +1,61 @@
+<h1 translate>Groups</h1>
+
+<div id="submenu">
+  <a ui-sref="users.group.create" os-perms="users.can_manage" class="btn btn-primary btn-sm" href="#">
+    <i class="fa fa-user-plus fa-lg"></i>
+    <translate>New</translate>
+  </a>
+  <a ui-sref="users.user.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview
+  </a>
+</div>
+
+<div class="row form-group">
+  <div class="col-sm-4">
+    <input type="text" os-focus-me ng-model="filter.search" class="form-control"
+        placeholder="{{'Filter'|translate}}">
+  </div>
+</div>
+
+<table class="table table-striped table-bordered table-hover">
+  <thead>
+    <tr>
+      <th><span ng-click="sortby='id';reverse=!reverse">
+            <translate>ID</translate>
+            <i class="fa" ng-show="sortby == 'id'"
+                ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+          </span>
+      <th><span ng-click="sortby='name';reverse=!reverse">
+            <translate>Name</translate>
+            <i class="fa" ng-show="sortby == 'name'"
+                ng-class="reverse ? 'fa-caret-down' : 'fa-caret-up'"></i>
+          </span>
+      <th os-perm="users.can_manage core.can_manage_projector" class="mini_width">Actions
+    </tr>
+  </thead>
+  <tbody>
+    <tr ng-repeat="group in groups | filter: filter.search | orderBy:sortby:reverse">
+      <td>{{ group.id }}
+      <td><a ui-sref="users.group.detail({id: group.id })">{{ group.name }}</a>
+      <td os-perms="users.can_manage core.can_manage_projector" class="nobr">
+        <!-- projector, TODO: add link to activate slidea-->
+        <a href="#TODO" os-perms="core.can_manage_projector" class="btn btn-default btn-sm"
+            title="{{'Show'|translate}}">
+          <i class="fa fa-video-camera"></i>
+        </a>
+        <!-- edit, TODO: add link -->
+        <a ui-sref="users.group.detail.update({id: group.id })" os-perms="users.can_manage"
+            class="btn btn-default btn-sm"
+            title="{{'Edit'|translate}}">
+          <i class="fa fa-pencil"></i>
+        </a>
+        <!-- delete -->
+        <a ng-click="delete(group)" os-perms="users.can_manage" class="btn btn-danger btn-sm"
+            title="{{'Delete'|translate}}">
+          <i class="fa fa-trash-o"></i>
+        </a>
+      </td>
```

Comment:
This tag can be removed, because:
```A th element's end tag may be omitted if the th element is immediately followed by a td or th element, or if there is no more content in the parent element```